ProjectLighthouse.Servers.Website/Pages/Login/BannedUserPage.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@page "/banned"
+@using LBPUnion.ProjectLighthouse.Localization.StringLists
+@model LBPUnion.ProjectLighthouse.Servers.Website.Pages.Login.BannedUserPage
+
+@{
+    Layout = null;
+    Model.Title = Model.Translate(ModerationStrings.BannedHeading);
+
+    string timeZone = (string?)ViewData["TimeZone"] ?? TimeZoneInfo.Local.Id;
+    TimeZoneInfo timeZoneInfo = TimeZoneInfo.FindSystemTimeZoneById(timeZone);
+}
+
+<div class="ui middle aligned left aligned">
+    <h1>@Model.Translate(ModerationStrings.BannedHeading)</h1>
+    <p>@Model.Translate(ModerationStrings.BannedExplain)</p>
+    <p>
+        <strong>@Model.Translate(ModerationStrings.CaseCreated):</strong>
+        @TimeZoneInfo.ConvertTime(Model.ModCase.CreatedAt, timeZoneInfo).ToString("M/d/yyyy @ h:mm tt") <br />  
+        @if (Model.ModCase.ExpiresAt != null)
```

Comment:
There should probably be something to have the date in a sentence using formatters. If you put arguments in the .resx file like {0}, {1}, etc. You can then call translate with multiple arguments that will be injected into the translatable string.